apps/io-services-cms-webapp/src/watchers/on-legacy-service-change.ts

```diff
@@ -89,6 +89,7 @@ const fromLegacyToCmsService = (
     status === "published" || status === "unpublished"
       ? "PublicationItemType"
       : "LifecycleItemType",
+  last_update_ts: Math.floor(Date.now() / 1000),
```

Comment:
**question:** if the change comes from the Legacy application, it shouldn't be `last_update_ts: service._ts`?

Reply:
done!